.github/workflows/build.yaml

```diff
@@ -77,6 +77,16 @@ jobs:
 
           cp ./scripts/livepeer-vmagent ./bin/livepeer-vmagent
 
```

Comment:
https://github.com/actions-rs/toolchain
```
      - uses: actions-rs/toolchain@v1
        with:
          toolchain: stable
```
might be needed before using `cargo`

Reply:
Thanks, trying it out!